pom.xml

```diff
@@ -442,6 +442,6 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <skipTests>false</skipTests>
         <groups></groups>
-        <logmanagerjar.name>LogManager</logmanagerjar.name>
+        <logmanagerjar.name>aws.greengrass.LogManager</logmanagerjar.name>
```

Comment:
Don't commit this change - it was good for testing but we don't need this to be part of this changes